src/app.ts

```diff
@@ -240,6 +240,10 @@ class TypescriptTextWriter implements ITypescriptTextWriter {
   public method(name: string, parameters: { parameter: string, type: string | TypescriptWriterCallback }[], returnType: string, singleLine = false) {
     this.writer.startIndentedLine(`${name}(`);
 
+    if (!singleLine) {
```

Comment:
Just some pretty-fying of the output, splits params onto newlines with trailing comma

Reply:
I think that we should give up any attempts to generate pretty output, because it'll mean that we should invent prettier. Lucky for us, it already exists :) I think we should generate ugly code and then just run it through prettier, based on DefinitelyTyped config.
Otherwise, LGTM :+1: